x-pack/plugins/spaces/kibana.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "spaces",
+  "version": "0.0.1",
+  "kibanaVersion": "kibana",
+  "configPath": [
+    "xpack",
+    "spaces"
+  ],
+  "server": true,
+  "ui": true
```

Comment:
not sure it should have `kibana.json` because the plugin is run by legacy platform. isn't it?

Reply:
Yep, this is an unused stub right now. I can remove if it helps clear things up

Reply:
I'd  rather remove it to avoid confusion